oidc-stub/src/main/kotlin/no/nav/modialogin/RedisRoutes.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.modialogin
+
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import redis.clients.jedis.JedisPool
+
+fun Application.redisIntrospection(host: String) {
+    val pool = JedisPool(host, 6379)
+    routing {
+        route("redis") {
+            get("keys") {
+                val keys = pool.resource.use { redis ->
+                    redis.auth("password123")
```

Comment:
skal dette passordet være "password123" og ikke hentes fra env?

Reply:
Vil ikke ha så mye å si, da det er en del av `oidc-stub` appen. Den blir brukt for å etterligne azureAd, openAM, unleash, og nå for å hente ut litt info fra redis (bare til testing).

Eneste grunnen til å flytte det til env er for å holde passordet samlet i docker-compose filen. Men da vil ikke appen kunne starte opp via intelliJ uten videre.

Reply:
ait, skjønner 🤠